src/button-group/index.d.ts

```diff
@@ -11,13 +11,6 @@ export interface STATE_CHANGE_TYPE {
   change: 'change';
 }
 
-export interface SIZE {
```

Comment:
Should we re-export the imported `SIZE` above?

Reply:
yeah, it's re-exported at the bottom of the file, just not visible in this changeset